markdown/extensions/codehilite.py

```diff
@@ -258,14 +266,16 @@ def code_unescape(self, text):
         text = text.replace("&amp;", "&")
         return text
 
-    def run(self, root):
+    def run(self, root: etree.Element) -> None:
         """ Find code blocks and store in `htmlStash`. """
         blocks = root.iter('pre')
         for block in blocks:
             if len(block) == 1 and block[0].tag == 'code':
                 local_config = self.config.copy()
+                text = block[0].text
+                assert text is not None
```

Comment:
Another assert....

Reply:
Another place where the code currently assumes that something is present there and violently errors if there's no match. mypy doesn't let it slide.

*TBD: It is worth checking whether this bug case can actually happen (in current code already). In any case we should be happy that mypy flagged this.*

---

With lines 275-278 reverted:

```console
$ mypy markdown
markdown/extensions/codehilite.py:276: error: Argument 1 to "code_unescape" of "HiliteTreeprocessor" has incompatible type "str | None"; expected "str"  [arg-type]
Found 1 error in 1 file (checked 33 source files)
```